appserver/web/gf-weld-connector/src/main/java/org/glassfish/weld/connector/WeldUtils.java

```diff
@@ -684,7 +694,7 @@ public static String getBeanDiscoveryMode(InputStream beansXmlInputStream) {
 
         if (beanDiscoveryMode == null) {
             //when empty beans.xml or bean-discovery-mode not specified
-            return "annotated";
+            return "all";
```

Comment:
I believe this should remain `"annotated"`, as that is the default value when attribute is not specified.